namespaces/live-1.cloud-platform.service.justice.gov.uk/justicedata-prod/resources/variables.tf

```diff
@@ -14,6 +14,10 @@ variable "namespace" {
   default = "justicedata-prod"
 }
 
+variable "domain" {
+  default = "data.justice.gov.uk"
```

Comment:
May be it needs re-name, can you please look in to 
https://ministryofjustice.github.io/technical-guidance/documentation/standards/naming-domains.html#naming-domains

Reply:
Had a conversation with @vijay-veeranki and @sivafirst . This is an existing domain where the nameservers will be deligated later. so the changes are fine.